src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/expressions/brgemm_copy_b_buffer_expressions.cpp

```diff
@@ -50,16 +50,17 @@ void RepackedWeightsBufferExpression::init_allocation_size(
     OPENVINO_ASSERT(brgemm_copy_b, "RepackedWeightsBufferExpression expects BrgemmCopyB as parent expression");
     const auto& brgemm_config = brgemm_copy_b->get_config();
 
-    const auto& in_subtensor = ov::snippets::utils::get_projected_subtensor(parent_expr->get_input_port(0));
+    const auto in_subtensor = ov::snippets::utils::get_projected_subtensor(parent_expr->get_input_port(0));
```

Comment:
Is change on this line expected?

Reply:
I noticed that `get_projected_subtensor` [returns](https://github.com/openvinotoolkit/openvino/blob/5d14a4a91551baaa6c648804d30cb2348cc782dc/src/common/snippets/src/utils/utils.cpp#L226) the object - so I decided to remove extra ref while I was updating this method.